packages/cobs/cobs.0.1/opam

```diff
@@ -0,0 +1,39 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Consistent Overhead Byte Stuffing (COBS) encoding and decoding"
+description:
+  "A lightweight Consistent Overhead Byte Stuffing (COBS) encoder and decoder written in pure OCaml with no dependencies"
+maintainer: ["Merlin04"]
+authors: ["Merlin04"]
+license: "MIT"
+homepage: "https://github.com/Merlin04/ocaml-cobs"
+doc: "https://github.com/Merlin04/ocaml-cobs"
+bug-reports: "https://github.com/Merlin04/ocaml-cobs/issues"
+depends: [
+  "ocaml"
```

Comment:
```suggestion
  "ocaml" {>= "4.08"}
  "ocaml" {with-test & >= "4.13"}
```
This should fix the issues by restricting the compiler. You can go the route you prefer